src/workers/parser/tests/__snapshots__/getSymbols.spec.js.snap

```diff
@@ -400,67 +400,67 @@ Array [
 exports[`Parser.getSymbols finds symbols in an html file 1`] = `
 Array [
   "functions:
- [(8, 2), (10, 3)]   sayHello name 
-[(22, 21), (24, 3)]   capitalize name 
-[(36, 3), (39, 3)]   iife  ",
+ [(9, 2), (11, 3)]   sayHello name 
```

Comment:
lol @jselmani @GarethSharpe we've gotta fix the CLRF [bug][b]

[b]:https://github.com/ryanjduffy/parse-script-tags/issues/3